dynet/cuda.cc

```diff
@@ -19,6 +19,13 @@ vector<Device*> initialize_gpu(DynetParams& params) {
   if (nDevices < 1)
     throw std::runtime_error("No GPUs found but DyNet compiled with CUDA support. Recompile without -DBACKEND=cuda");
 
+  // Get CUDA runtime/driver version
+  int runtimeVersion = 0;
+  int driverVersion = 0;
+  cudaRuntimeGetVersion(&runtimeVersion);
+  cudaDriverGetVersion(&driverVersion);
+  cerr << "[dynet] CUDA driver/runtime version is "<<runtimeVersion/1000<<"."<<(runtimeVersion%100)/10<<"/"<<driverVersion/1000<<"."<<(driverVersion%1000)/10<<endl;
```

Comment:
Nitpick: "CUDA driver/runtime version is" -> "CUDA driver/runtime versions are"

Also can you split in multiple lines and add spaces so that it is more readable?

Reply:
looks like this?
[dynet] CUDA driver version is ...
[dynet] CUDA runtime version is ...

Reply:
No I mean split the code in different lines